import/src/test/java/com/google/cloud/healthcare/imaging/dicomadapter/CFindServiceTest.java

```diff
@@ -134,7 +134,6 @@ public void basicCFindServiceTest(IDicomWebClient serverDicomWebClient,
     // Issue CFIND
     DimseRSPAssert rspAssert = new DimseRSPAssert(association, expectedStatus);
     association.cfind(
-        UID.StudyRootQueryRetrieveInformationModelFIND,
```

Comment:
Why did this change?

Reply:
Dcm4che version update, signatures of few methods changed.